cranelift/filetests/filetests/isa/riscv64/simd-iadd.clif

```diff
@@ -0,0 +1,73 @@
+test compile precise-output
+set unwind_info=false
+target riscv64 has_v
+
+
+function %iadd_i8x16(i8x16, i8x16) -> i8x16 {
+block0(v0: i8x16, v1: i8x16):
+    v2 = iadd v0, v1
+    return v2
+}
+
+; VCode:
+; block0:
+;   vadd.vv v10,v11,v10 #avl=16, #vtype=(e8, m1, ta, ma)
+;   ret
+; 
+; Disassembled:
+; block0: ; offset 0x0
+;   .byte 0x57, 0x70, 0x08, 0xcc
+;   .byte 0x57, 0x05, 0xb5, 0x02
```

Comment:
We're on capstone 0.9 right now and it looks like the latest is 0.11, want to see if updating it would enable disassembling these instructions? If so I think it's reasonable to try to get in a `cargo vet` entry or an exemption since it's purely used during tests.

Reply:
I tried to upgrade and rerun the tests, but It doesn't seem to have helped. I also looked in the docs for capstone to check if we needed to enable some additional feature, but I only found one for the [Compressed extension](https://docs.rs/capstone/latest/capstone/enum.ExtraMode.html#variant.RiscVC). So it looks like It still doesn't support it which is really unfortunate.

Reply:
Ok now I'm a bit curious. Feel free to disregard this comment in the context of this PR as it's of course fine to have what's here and there's no need to deal with this for this PR.

First question though: how up-to-date is capstone in the Rust bindings? According to [this](https://github.com/capstone-rust/capstone-rs/tree/master/capstone-sys/capstone/) the latest commit there is to update to https://github.com/capstone-engine/capstone/commit/3b2984212fe which is almost a year-old at this point.

...


Ok nevermind answered my own question. I saw `include/riscv.h` in the Rust bindings but didn't see it on the `main` branch of the `capstone` repository. As the commit link above shows, though, there's a `next` branch which it looks like the Rust bindings are drawing from which does indeed have `include/riscv.h`. A quick scan of that header and I don't think there's V-related instructions, or at least I don't see a huge wad of `V`-prefixed instructions in that header, so it's likely that capstone itself doesn't have support for this yet.

Reply:
When turning on Capstone disassembly in precise-output filetests recently, we discussed that Capstone isn't particularly up to date on most targets. Its main advantage is that it supports a lot of targets. I believe consensus was that we'd be happy to switch disassemblers but nobody wants to spend time investigating alternatives. Also I think we don't know of any one disassembler that supports all our targets, although it wouldn't be awful to use a different disassembler for each target, either.